File: src/main/java/com/wikia/tibia/repositories/AchievementRepository.java
package com.wikia.tibia.repositories;

import com.wikia.tibia.gateways.AchievementGateway;
import com.wikia.tibia.objects.Achievement;

public class AchievementRepository extends WikiObjectRepository {

    public AchievementRepository() {
        super(Achievement.class, new AchievementGateway());
    }
}

File: src/main/kotlin/com/wikia/tibia/repositories/AchievementRepository.kt
package com.wikia.tibia.repositories

import com.wikia.tibia.enums.Contract
import com.wikia.tibia.gateways.WikiObjectGateway
import com.wikia.tibia.objects.Achievement

class AchievementRepository : WikiObjectRepository<Achievement>(
    wikiObjectClass = Achievement::class.java,
    wikiObjectGateway = WikiObjectGateway(Contract.ACHIEVEMENTS)
)
